### ISSUE TITLE
Move groupOrder in patch_files.py

### ISSUE THREAD

**Issue by Sharlikran** [wrye-bash/wrye-bash#287]
```Python
# Parent records must be processed before any children
# EYES,HAIR must be processed before RACE
```
The comments for the set in [patch_files.py](https://github.com/wrye-bash/wrye-bash/blob/dev/Mopy/bash/patcher/patch_files.py#L424-L436) imply that Wrye has to process child and parent records in a specific order.  I'd like this set to be moved to constants.py or `__init__.py` so it can be updated properly.  There is also no way to know if this is currently impacting Skyrim's enabled patchers but since the list is for Oblivion only it is missing some of Skyrim's records.  Until it is moved and tested at a later time there is no way to gauge the impact it will have on the patcher.

**Comment by Sharlikran** [comment]
Not part of the issue exactly but can Wrye access the game specific listTypes that early?  Because [leveledLists](https://github.com/wrye-bash/wrye-bash/blob/dev/Mopy/bash/patcher/patch_files.py#L445) should be looking listTypes from the [game\__init__.py](https://github.com/wrye-bash/wrye-bash/blob/dev/Mopy/bash/game/skyrim/__init__.py#L1267) file.

**Comment by Utumno** [comment]
Yes whatever is in (or imported in) game/game/init.py is available by the time buildPatch runs (a general overhaul of booting plus docs for what is available when is on the TODOs for the rest 20% of refactoring that remains). Note however that this is the CBash one and it needs a set as opposed to a tuple apparently. Thanks for managing issues btw ! I see you read the labels wiki I was at pains to write - turns out it is useful after all :)

**Comment by Sharlikran** [comment]
I didn't know that was for CBash only.  Is there one for PBash?

**Comment by Utumno** [comment]
I mean the line [leveledLists](https://github.com/wrye-bash/wrye-bash/blob/dev/Mopy/bash/patcher/patch_files.py#L445) is inside the `CBash_PatchFile.buildPatch()` method.

**Comment by Utumno** [comment]
Marking as TODO as it should not appear in the second posts - it is basically an internal code issue. Will get back to it

**Comment by Infernio** [comment]
This is indeed CBash-only, the PBash equivalent is `RecordHeader.topTypes`. No idea why `groupOrder` is different from that one, but I can tell you that `topTypes` is correct, since it matches the [group order defined by xEdit](https://github.com/TES5Edit/TES5Edit/blob/081cd8672bd366306eaafbbc8771f3a72c6cea7a/wbDefinitionsTES4.pas#L5179-L5235).
The point in PBash's code where the order of `topTypes` becomes important is `parsers.ModFile.save`, where it determines in which order to write out the record groups:
https://github.com/wrye-bash/wrye-bash/blob/545cad09d29cf4b17ed470caba974829f5876d7f/Mopy/bash/parsers.py#L4053-L4056

**Comment by Infernio** [comment]
Closing CBash issue as `R-wont-fix` (#530).
